src/axom/sidre/tests/sidre_group.cpp

```diff
@@ -2729,6 +2729,55 @@ TEST(sidre_group, save_load_preserve_contents)
   DataStore::setConduitDefaultMessageHandlers();
 }
 
+//------------------------------------------------------------------------------
+TEST(sidre_group, save_layout_protocols)
+{
+  // Tests calls to save using sidre_layout_json and conduit_layout_json
+  // protocols. No loads are attempted as these protocols are used for
+  // save only.
+
+  DataStore::setConduitSLICMessageHandlers();
+
+  const std::string file_path_base("sidre_save_layout_protocols.");
```

Comment:
Suggestion: This might be slightly clearer if the `.` were in the extension below instead of the path prefix here